launch_parser_on_nidm.py

```diff
@@ -26,76 +42,57 @@ def main():
     """
 
     parser = argparse.ArgumentParser()
-
     parser.add_argument("--input_dir", type=str, default="nidmresults-examples",
                         help="data dir where .m and .html are researched")
     parser.add_argument("--output_dir", type=str, default="examples",
                         help="output dir where results are written")
     parser.add_argument("--verbose", action="store_true", help="more print")
-
     opt = parser.parse_args()
 
-    if os.path.exists(opt.output_dir):
-        shutil.rmtree(opt.output_dir)
-    os.makedirs(opt.output_dir, exist_ok=True)
+    # fix the seed to have identical jsonlds if there are no other modifications
+    random.seed(1)
```

Comment:
The value of the seed (=1) is different from the one used in https://github.com/Inria-Empenn/BEP028_BIDSprov/blob/40e6883693b0ac682d722993ef1c308283fc921e/bids_prov/tests/test_spm_parser.py   at line :   [random.seed(14)  # Control random generation for test, init at each import](https://github.com/Inria-Empenn/BEP028_BIDSprov/blob/40e6883693b0ac682d722993ef1c308283fc921e/bids_prov/tests/test_spm_parser.py#L15)

A seed value to 14 should be more consistency ?